tests/e2e.rs

```diff
@@ -1,17 +1,17 @@
 #![cfg(test)]
+#![allow(clippy::expect_fun_call)]
```

Comment:
Should add this to the clippy config, and add it to our company-wide clippy config, if we don't like that lint.

Reply:
I suggest to create an issue out of this and leave it explicitly in the test file for this PR.

Reply:
This has a (small) performance impact. While this isn't important in tests, there's no way to tell clippy that we don't care about it in tests specifically, so I just fixed it instead.

We can raise an issue to discuss whether we want to trade-off this bit of performance for legibility separately.